chanjo/calculate.py

```diff
@@ -30,3 +31,23 @@ def gene_metrics(self, *genes):
                      .filter(Transcript.gene_id.in_(genes))
                      .group_by(Transcript.gene_id))
         return query
+
+
+    def omim_coverage(self, samples):
+        """Calculate coverage for OMIM panel."""
+        sample_ids = [sample.id for sample in samples]
+        query = self.query(
+            TranscriptStat.sample_id.label('sample_id'),
+            func.avg(TranscriptStat.mean_coverage).label('mean_coverage'),
+            func.avg(TranscriptStat.completeness_10).label('mean_completeness'),
+        ).join(
+            TranscriptStat.transcript,
+        ).filter(
+            TranscriptStat.sample_id.in_(sample_ids),
+            Transcript.gene_id.in_(OMIM_GENE_IDS),
```

Comment:
Have you checked the performance for this query? Feels like it can take some time to calculate?